src/local/motion/NonRigidObjectTransformationUsingBSplines.cxx

```diff
@@ -340,6 +404,46 @@ NonRigidObjectTransformationUsingBSplines()
   this->set_defaults();
 }
 
+template <int num_dimensions, class elemT>
+NonRigidObjectTransformationUsingBSplines<num_dimensions,elemT>::
+NonRigidObjectTransformationUsingBSplines(std::string filename_x, std::string filename_y, std::string filename_z, int &bspline_order)
+{
+    this->set_defaults();
+
+    // Get the origin and spacing from the x and assume it's the same for y and z
+    shared_ptr<DiscretisedDensity<3,float> > image_sptr(read_from_file<DiscretisedDensity<3,float> >(filename_x));
+
+    if (is_null_ptr(image_sptr))
+        warning("Error reading %s", filename_x.c_str());
```

Comment:
`error` not `warning` in all of these

Reply:
Some return `Succeeded::no`, so that's why I didn't throw an error. But ok, I'll do that.